pkgs/development/python-modules/penelope/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
```

I think this tends to be viewed as more idiomatic
```suggestion
{ lib
, stdenv
, buildPythonPackage
, fetchFromGitHub
}:
```